src/server/api/user/index.ts

```diff
@@ -43,11 +44,19 @@ const validator = createValidator({ passError: true })
  */
 function preprocessPotentialIncomingFile(
   req: Express.Request,
-  _: Express.Response,
+  res: Express.Response,
   next: Express.NextFunction,
 ) {
   if (req.files) {
     req.body.files = req.files
+    if (req.body.tags) {
```

Comment:
nit: this will still be parsed successfully even if the  tag is not an array of string! The error will be caught by the validators (one of the subsequent middlewares)

Reply:
yup, that sounds right to me! the validating of the exact format should probably be left to the Joi schema later on, as I understood this to be just for the preprocessing step - would you have any thoughts on how to improve this? or perhaps adding comments would help to clarify the intention here?

Reply:
alright makes sense!